features/step_definitions/cli.rb

```diff
@@ -183,6 +183,16 @@
     })
 end
 
+Given /^I wait(?: up to #{NUMBER} seconds)? for the last background process to finish$/ do |seconds|
+  seconds = Integer(seconds) rescue 60
+  success = wait_for(seconds) {
+    if @bg_processes.last.finished?
+      true
+    end
+  }
```

Comment:
you can simplify by replacing the 3 lines with `@bg_processes.last.finished?`. The `if` is redundant here.